packages/cashier/src/pages/account-transfer/account-transfer-form/__tests__/account-transfer-form-side-note.spec.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import AccountTransferNote, { TAccountTransferNoteProps } from '../account-transfer-form-side-note';
+
+let mockProps: TAccountTransferNoteProps;
```

Comment:
please use snake_case for variables. Camel case is used for functions.
```suggestion
let mock_props: TAccountTransferNoteProps;
```